scripts/cp_msi_app_update.js

```diff
@@ -0,0 +1,6 @@
+// Copyright (c) 2016-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+const fs = require('fs');
+fs.copyFileSync('release/win-unpacked/resources/app-update.yml', 'release/msi-app-update.yml');
+fs.unlinkSync('release/win-unpacked/resources/app-update.yml');
```

Comment:
Two nitpicks, probably unneeded.
- Does it make sense to put these routes in a constant? The first one is repeated twice in this file, and I guess the second one will be used somewhere else, but maybe not in a JS environment...
- Do we want to check any possible error on the copy before trying to unlink the original file?